lib/wallet/src/Cardano/Wallet/Delegation.hs

```diff
@@ -172,10 +175,11 @@ joinStakePoolDelegationAction
     -> Set PoolId
     -> PoolId
     -> PoolLifeCycleStatus
-    -> IO Tx.DelegationAction
+    -> Bool
+    -> IO (Tx.DelegationAction, Maybe Tx.VotingAction)
 joinStakePoolDelegationAction
     tr DBLayer{..} currentEpochSlotting
-        knownPools poolId poolStatus = do
+        knownPools poolId poolStatus inConway = do
```

Comment:
```suggestion
        knownPools poolId poolStatus era = do
```

When rebasing on #4477, please pass a value of type `era :: Write.RecentEra era` to the pure function instead.